dataland-frontend/src/components/forms/parts/elements/basic/ExtendedDataPointFormField.vue

```diff
@@ -103,16 +127,21 @@ export default defineComponent({
     fileReferenceAccordingToName(): string {
       return getFileReferenceByFileName(this.currentReportValue, this.injectReportsNameAndReferences);
     },
+    showDataPointFields(): boolean {
+      return !this.shouldBeToggle || this.dataPointIsAvailable;
```

Comment:
"shouldBeToggle" seems like an unclear name to me in this context.
maybe "isDataPointToggleable"

Reply:
It has been changed to "isDataPointToggleable"